test/e2e/windows/runtime.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package windows
+
+import (
+	"fmt"
+	"path"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/kubernetes/test/e2e/common"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = framework.KubeDescribe("[sig-windows] Container Runtime", func() {
+	f := framework.NewDefaultFramework("container-runtime")
+
+	BeforeEach(func() {
+		// NOTE(claudiub): These tests are Windows specific.
+		framework.SkipUnlessNodeOSDistroIs("windows")
+	})
+
+	Describe("blackbox test", func() {
+		Context("when starting a container that exits", func() {
```

Comment:
How does this differ from e2e/common/runtime and why?

Reply:
It does not create the Pod with SELinuxOptions in the PodSecurityContext.

This test might become obsolete, if the following PR won't get reverted: https://github.com/kubernetes/kubernetes/pull/70092

If the mentioned PR won't get reverted by next week, I will remove this test case.